ts/components/wallet/card/CardComponent.style.ts

```diff
@@ -30,6 +30,10 @@ export default StyleSheet.create({
     justifyContent: "flex-start"
   },
 
+  row: {
+    flexDirection: "row"
```

Comment:
I see that if I go, here on the browser, on "file changed" tab

Reply:
Because you have an outdated master locally? Try a pull the code is in master here https://github.com/teamdigitale/io-app/blob/master/ts/components/wallet/card/CardComponent.style.ts#L33